package.json

```diff
@@ -84,6 +84,7 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
     "helmet": "^2.1.2",
+    "html-loader": "^0.4.4",
```

Comment:
@ZeroCho has spoken! True to our aim of going trimming this repo! Alright! Will remove it in the next upcoming commit.